frame/wormhole/backing/s2s/src/lib.rs

```diff
@@ -259,6 +285,14 @@ pub mod pallet {
 			};
 			let amount = token_info.value.ok_or(<Error<T>>::InvalidTokenAmount)?;
 
+			// Make sure the total transfered is less than daily limited
+			let unlocked = Self::get_daily_unlocked();
+			let limited = <DailyLimited<T>>::get();
+			ensure!(
+				limited.saturating_sub(unlocked) >= amount.low_u128().unique_saturated_into(),
+				<Error<T>>::RingDailyLimited
```

Comment:
If error happens, the token in issuing chain will be return back to user who did redeem right?

Reply:
Yes